hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/ringbuffer/RingbufferAddAllMessageTask.java

```diff
@@ -76,7 +76,7 @@ public Permission getRequiredPermission() {
 
     @Override
     public String getMethodName() {
-        return "addAllAsync";
```

Comment:
Can this lead to security problems? Because one can't control the async methods independently. 

To me it doesn't make sense to provide granularity on such a level; but I'm wondering.